packages/Avatar/src/Avatar.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import * as sc from "./Avatar.styles";
+
+const propTypes = {
+  /** Avatar content. It can be initial as a string or icon */
+  children: PropTypes.node,
+  /** Background color of the Avatar */
+  backgroundColor: PropTypes.string.isRequired,
+  /** Color for the initial or icon */
+  color: PropTypes.string.isRequired,
+  /** Size of Avatar */
+  size: PropTypes.oneOf(ShirtSizes.LIMITED),
+};
+
+const defaultProps = {
+  children: null,
+  size: "medium",
+};
+
+function Avatar(props) {
+  const { backgroundColor, children, size, color } = props;
+
+  const styleProps = {
+    backgroundColor,
+    size,
+    color,
+  };
+
+  return (
```

Comment:
https://codesandbox.io/s/avatar-inrdr?file=/src/App.js look at this example you don't need to split the props. :)